cipher_suite.go

```diff
@@ -68,6 +72,16 @@ type cipherSuite interface {
 	decrypt(in []byte) ([]byte, error)
 }
 
+// CipherSuiteName provides the same functionality as tls.CipherSuiteName
+// that appeared first in Go 1.14
+func CipherSuiteName(id uint16) string {
```

Comment:
I kept the signature identical to stdlib, but I'm thinking that it might make more sense to make this `func CipherSuiteName(id CipherSuiteID) string` since we require the use of a `CipherSuiteID` everywhere instead of a `uint16`.

Reply:
I don't have strong opinion on this, but using `CipherSuiteID` with a comment about type difference maybe fine.

Reply:
Yap, I've changed it. I think it makes more sense that way.